spec/unit/lib/web/page_spec.rb

```diff
@@ -178,6 +195,7 @@
       subject { described_class.expanded_url }
       before do
         allow(described_class).to receive(:url_template) { '/users' }
+        described_class.class_eval { root_url 'http://my.website.com' }
```

Comment:
root_url is specific case. Do we need the specific case here or we need to test with general app_url?